go/sns/README.md

```diff
@@ -0,0 +1,101 @@
+# AWS SDK for Go code examples for Amazon SNS
+
+## Purpose
+
+These examples demonstrate how to perform several Amazon Simple Notification Service (Amazon SNS) operations.
+
+## Prerequisites
+
+You must have an AWS account, and have your default credentials and AWS Region
+configured as described in
+[Configuring the AWS SDK for Go](https://docs.aws.amazon.com/sdk-for-go/v1/developer-guide/configuring-sdk.html)
+in the _AWS SDK for Go Developer Guide_.
+
+## Running the code
+
+### MakeTopic/MakeTopic.go
+
+This example creates an Amazon SNS topic.
+
+`go run MakeTopic.go -t TOPIC`
+
+- _TOPIC_ is the name of the topic to create.
+
+The unit test mocks the service client and the `CreateTopic` function.
+
+### PublishMessage/PublishMessage.go
+
+This example publishes a message to an Amazon SNS topic.
+
+`go run PublishMessage.go -m MESSAGE -t TOPIC-ARN`
+
+- _MESSAGE_ is the message to publish.
+- _TOPIC-ARN_ is the ARN of the topic to which the user subscribes.
+
+The unit test mocks the service client and the `Publish` function.
+
+### ShowSubscriptions/ShowSubscriptions.go
+
+This example displays the ARNs of your Amazon SNS subscriptions.
```

Comment:
...the **Amazon Resource Names (ARNs)...

Reply:
Fixed